keyboards/winry/winry315/rules.mk

```diff
@@ -11,5 +11,6 @@ BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 AUDIO_ENABLE = no           # Audio output
 ENCODER_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
```

Comment:
This should not be enabled at the keyboard level, and should be moved to `keymaps/*/rules.mk`.

Reply:
I've made this change.